framework/build.xml

```diff
@@ -362,6 +368,52 @@
         </antcall>
     </target>
 
+    <target name="signature-inference-tests" depends="jar,build-tests"
```

Comment:
Recommend adding the deletes in this target to the "clean" target as well, in case this target was run at some point and interrupted in the middle.